.github/workflows/publish-sdist-wheels.yml

```diff
@@ -62,14 +62,23 @@ jobs:
     name: Build sdist
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
       - name: Set up Python 3.6
         uses: actions/setup-python@v2
         with:
           python-version: 3.6
       - name: Build a source tarball
         run:
           python setup.py sdist
+      - name: Test tarball
+        run: |
+          sudo apt install -y libhdf5-dev
+          pip install -r tests/requirement_tests.txt
+          pip install dist/*
+          nosetests -s -v -P tests
```

Comment:
Stupid question :
run from actions propagates the returned "successful or not termination" of the command line ?

Reply:
Yes. https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions#about-exit-codes

Reply:
Au top topper !